docs/rst/enhancements/easy_mode/easy_mode.rst

```diff
@@ -0,0 +1,138 @@
+.. _easy_mode:
+
+Easy Mode
+=========
+
+The new ``Vulcanexus Easy Mode`` aims to simplify, enhance and optimize the deployment of any ROS 2 application reinforcing the overall out-of-the-box user experience.
+Moreover, it claims to be the default discovery behavior within :ref:`Vulcanexus <vulcanexus_introduction>`.
+This section reveals its significance, operating mode and impact.
+
+Quick Overview
+^^^^^^^^^^^^^^
+
+The ``Vulcanexus Easy Mode`` is a builtin discovery mode that simplifies to the most the deployment of ROS 2 applications with the use of `Discovery Server <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html>`_.
+To enable this feature the user only needs to set the environment variable ``EASY_MODE=<ip>``.
+This ``<ip>`` can be either the IP address of the current host or the ``<ip>`` of an external host in the same LAN acting as the central point of discovery (``master``).
+A new discovery server will be spawned in background in the domain specified by the ``ROS_DOMAIN_ID`` environment variable.
+The following diagram illustrates the concept:
+
+.. image:: ../../figures/enhancements/easy_mode/easy_mode_quick_intro.png
```

Comment:
I'm missing a legend with the meaning of the arrows (discovery and user data). I'd also remove the Domain from the image as may be confusing for ROS 2 users as they are not used to using domains.